### PR TITLE
Gradio blocks competition event card v2

### PR DESCRIPTION
**Pull request by FarukOzderim**
Creating this PR to enable collaboration on the repo. You can create PRs onto this branch

History: #27

### CODE REVIEW

File: gradio-blocks/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+## Event Plan
+
+In this event, you will:
+
+1. Learning about Gradio and the new Blocks Feature
+2. Building your own Blocks demo using Gradio and Hugging Face Spaces
+3. Submitting your demo on Spaces to the Gradio Blocks Party Organization
+4. Share your blocks demo with a permanent shareable link 
+5. Win Prizes
+
```
Comment (FarukOzderim) [inline gradio-blocks/README.md:53]:
```suggestion
Participants will be building and sharing Gradio demos using the Blocks feature. We will share a list of ideas of spaces that can be created using blocks or participants are free to try out their own ideas. At the end of the event, spaces will be evaluated for creativity and prizes will be given. 

### To participate in the event

- Join the organization for Blocks event
    - [https://huggingface.co/Gradio-Blocks](https://huggingface.co/Gradio-Blocks)
- Join the discord
    - [discord](https://discord.com/invite/feTf9x3ZSB)
```

File: gradio-blocks/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+- Join the organization for Blocks event
+    - [https://huggingface.co/Gradio-Blocks](https://huggingface.co/Gradio-Blocks)
+- Join the discord
+    - [discord](https://discord.com/invite/feTf9x3ZSB)
+
+
+Participants will be building and sharing Gradio demos using the Blocks feature. We will share a list of ideas of spaces that can be created using blocks or participants are free to try out their own ideas. At the end of the event, spaces will be evaluated for creativity and prizes will be given. 
+
+
+potential ideas for creating spaces:
```
Comment (FarukOzderim) [inline gradio-blocks/README.md:77]:
```suggestion
## Potential ideas for creating spaces
```

**Review by FarukOzderim — COMMENTED** [review]
Some positional change recomms

**Comment by FarukOzderim** [comment]
A few suggs are above @AK391

**Comment by osanseviero** [comment]
Is this built on top of https://github.com/huggingface/community-events/pull/27? If yes, maybe you can change the branch to which this PR is getting merged so it's easier to see the diff

**Comment by AK391** [comment]
@FarukOzderim @osanseviero if it is easier to merge can we go back to #27 , I also dont see the changes made here reflecting on my branch

**Comment by FarukOzderim** [comment]
The issue with #27 is it is a fork, and cannot take PRs as a destination, moved that fork branch to this branch for the sake of ease of collab.

@AK391 could you try cloning this repo instead of your fork and switching out to this branch?

**Comment by FarukOzderim** [comment]
@AK391 @osanseviero if you are good with this PR, let's merge it?

**Comment by AK391** [comment]
@FarukOzderim sounds good, I made a few more changes to https://huggingface.co/spaces/Gradio-Blocks/README, wanted to make this in sync with that, for example the dalle-mini demo is now a different video

**Comment by AK391** [comment]
@FarukOzderim sounds good closing

**Pull request closed without merging**